mailbox/api/src/main/java/org/apache/james/mailbox/MessageIdManager.java

```diff
@@ -32,7 +32,7 @@
 
 public interface MessageIdManager {
 
-    void setFlags(Flags newState, FlagsUpdateMode replace, MessageId messageId, MailboxId mailboxId, MailboxSession mailboxSession) throws MailboxException;
+    void setFlags(Flags newState, FlagsUpdateMode replace, MessageId messageId, MailboxSession mailboxSession) throws MailboxException;
```

Comment:
So can you add back the MailboxId parameter (or MailboxIds if you want) to be more consistent with MessageManager API and IMAP?